Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
-import './Display.css';
+import './Display.scss';
+
+const CalculatorDisplay = () =>{
+    return (
+        <div className = 'display'>0</div>
+
+    )
```

Comment:
In the future (when you have time to add functionality to this calculator React app), you'll want to change the `0` value to a prop holding the current display value for your calculator (perhaps something like `props.calcDisplayText`).